.github/map_checker/matchers.yml

```
@@ -7,6 +7,11 @@
   - "RMCAdminFax"
   - "AdminPDA"
   - "AdminHypo"
+  - "RMCPlushieAlec"
+  - "RMCPlushieNyx"
+  - "RMCPlushieXippy"
+  - "RMCPlushieProvost"
+  - "RMCPlushieCrow"
```

Missing the new one unless you want it to be mapped
```suggestion
  - "RMCPlushieCrow"
  - "RMCPlushieMotivationalMoth"
```

---

I forgor